cmd/evmosd/root.go

```diff
@@ -125,7 +125,7 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		NewTestnetCmd(app.ModuleBasics, banktypes.GenesisBalancesIterator{}),
 		debug.Cmd(),
 		config.Cmd(),
-		pruning.PruningCmd(a.newApp),
+		pruning.Cmd(a.newApp, app.DefaultNodeHome),
```

Comment:
Not sure if removing the `pruning=nothing` etc. option is a problem for some validators? This flag is only available for `PruningCmd` and not `Cmd`

Reply:
these flags just overrides whatever the config is on the `app.toml` file.
It's no big impact removing this flag